apps/docs/guide/deploying/self-hosted.md

```diff
@@ -0,0 +1,164 @@
+
+The server minimum requirements will depend on the expected usage of the application and on the specifications of the Cartesi machine in use, like its RAM size and total size. The developer will have to experiment with different configurations to find the best fit for his needs. We suggest starting with a minimum of 8GB of RAM, and scaling up vertically as needed.
+
+The Cartesi rollups node is distributed as a Docker image, so the server will need to have Docker installed. Any popular cloud provider, like AWS, GCP, Azure, Digital Ocean, or Linode, is capable of running docker containers, either manually by spawing a server and running docker, or by using a managed container infrastructure like Kubernetes.
+
+The developer can also use a service like [Fly.io](https://fly.io) to deploy the application node.
+
+We will describe below two methods of running a rollups node, the first using Docker, which can be used as the basic information to use any cloud provider, and the second using [Fly.io](https://fly.io), a managed service for running Docker containers.
+
+### Using Docker in any cloud provider
+
+Assuming the developer completed the above requirements, and has a complete `.env` file, he can run the rollups node using the following command:
+
+```shell
+$ docker run --env-file <env-file> -p 10000:10000 <image-id>
+(...)
+INFO http: listening at [::]:10000
+(...)
+INFO rollups-node: all services are ready
+(...)
+```
+
+In the examples above the `<env-file>` file is `0xc87999b8a93609268b10de25f2e49d35f80fad92813310edc585ed644a9805d3.env` and the `<image-id>` is `sha256:5c355a9bddc92aa08987f395a257a0b32a51552c969eb161386e46f9380ea2ac`.
+
+The developer is free to use any managed container solution, like Kubernetes. The rollups node is just a single container, with a single exposed port, that must be accessible from the internet.
+
+### Using fly.io
+
+[Fly.io](https://fly.io) is platform as a service that allows developers to easily deploy applications packaged as Docker containers. The rollups node can be deployed to Fly.io. Choose an `<app-name>` and follow the steps below:
+
+1. [Install the flyctl CLI](https://fly.io/docs/hands-on/install-flyctl/)
+2. [Create an account and login](https://fly.io/docs/hands-on/sign-up-sign-in/)
+3. Create an application:
+
+    ```shell
+    $ fly app create <app-name>
+    New app created: <app-name>
+    ```
+
+4. Create a postgres database application:
+
+    ```shell
+    fly postgres create --initial-cluster-size 1 --name <app-name>-database --vm-size shared-cpu-1x --volume-size 1
+    ```
+
+5. Attach database to the node application:
+
+    The previous step should have printed the connection string for the database. The developer can use it to configure the `CARTESI_POSTGRES_ENDPOINT` environment variable.
```

Comment:
Why is this sentence here? I understand that the code to attach has nothing to do with this connection string, right? Maybe we should place this sentence at the end of step 4 (e.g., "take note of the connection string to use it later on")